.github/workflows/e2e-tests-all.yml

```diff
@@ -0,0 +1,48 @@
+name: e2e tests all
+
+on: [push]
+
+jobs:
+  e2e-tests-all:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup npm cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install dependencies (root)
+        run: npm ci
+        env:
+          CI: true
+
+      - name: Run e2e tests (all)
+        uses: cypress-io/github-action@v1
+        with:
+          start: npm run e2e:prod
```

Comment:
The three `e2e` test files are identical except for this line. Is it possible to use a single file and just change the command that will be executed? Maybe using 

```yaml
strategy:
  matrix:
    test-command: [prod, prod:module, prod:es5]
```
and then, in `steps`

```yaml
- name: Run e2e tests (es5)
  uses: cypress-io/github-action@v1
  with:
    start: npm run e2e:${{ matrix.test-command }}
```

like in `unit-tests`, or something like that?

I know nothing about GitHub Actions, BTW. 🙈

Reply:
Maybe it can be done with `ci_node_total` and `ci_node_index` like [this guy does for cypress](https://gist.github.com/ArturT/3535fc24bf2fac577462b9162c12c1d6), but it is not documented anywhere. I'll give it a try.

Reply:
It worked using `ci_node_total` and `ci_node_index` https://github.com/frontity/frontity/pull/244/commits/f4b10d0d300fc51e0c5e436339cff2b05036bf5f :)

Thanks for the suggestion @DAreRodz!